discover/discover_test.go

```diff
@@ -5,11 +5,12 @@ import (
 	"os"
 	"testing"
 
+	"github.com/effxhq/effx-cli/data"
 	"github.com/effxhq/effx-cli/discover"
 	"github.com/stretchr/testify/require"
 )
 
-func Test_Discover_Services(t *testing.T) {
+func Test_Discover_Services_From_Yaml(t *testing.T) {
```

Comment:
Can you add tests to our CI? It's a shame that we aren't running those tests automatically.